packages/app-builder/public/locales/en/settings.json

```diff
@@ -23,6 +23,19 @@
   "users.inbox_user_role.member_count_other": "MEMBER in {{count}} inboxes",
   "users.inbox_user_role.unknown": "unknown",
   "api_keys": "API Keys",
+  "api_keys.new_api_key": "New API Key",
+  "api_keys.copy_api_key": "Make sure to copy it now as you will not be able to see this again.",
+  "api_keys.description": "Description",
+  "api_keys.role": "API key role",
+  "api_keys.role.admin": "ADMIN",
+  "api_keys.role.publisher": "PUBLISHER",
+  "api_keys.role.builder": "BUILDER",
+  "api_keys.role.viewer": "VIEWER",
+  "api_keys.role.api_client": "API_CLIENT",
+  "api_keys.role.unknown": "UNKNOWN",
+  "api_keys.create": "Create new API Key",
+  "api_keys.delete": "Delete API Key",
+  "api_keys.delete.content": "You are about to delete this API key. This action is irreversible, do you want to proceed?",
```

Comment:
same as in backend, I have a preference for "revoke" rather than "delete" (but that's just wording of course)